fs.go

```diff
@@ -926,6 +1019,22 @@ var archiveExtensions = []string{
 	".tar.lz",
 }
 
+// PathIsArchive returns true if the path ends with an archive file (i.e.
+// whether the path traverse to an archive) solely by lexical analysis (no
+// reading the files or headers is performed).
+func PathIsArchive(path string) bool {
```

Comment:
Does this have to be exported?
```suggestion
func pathIsArchive(path string) bool {
```
(also, the name is a bit confusing, since it actually returns true when the path refers to a file _within_ an archive...)

Reply:
> since it actually returns true when the path refers to a file within an archive...

Hrmm that shouldn't be the case unless there's a bug in the new function (or I misunderstood what you meant there).

I understood the above to be what `PathContainsArchive` does, but `PathIsArchive` should only return true for stuff like:

```
# Things where the file path is reffering to an archive 
/foo/bar/baz.tar.gz
/foo/bar/baz.tgz
```

Not stuff like:

```
/foo/bar/baz.tar.gz/sub/file
```

> Does this have to be exported?

Nope! I just figured it was a useful function similar to `PathContainsArchive`, but I can also make it private if you prefer ^\_^

--

Based on the above, let me know if you'd like to to go ahead and make it private and/or change the name. Happy to do either :ok_hand:

Reply:
You're right, my bad.